app/src/types/event.ts

```diff
@@ -79,16 +64,39 @@ export const validateEvent = (
   };
 };
 
-export type IWriteModel = IEventContract;
-export type IViewModel = WithId<IWriteModel>;
-
-export const serializeEvent = (event: IEvent): IWriteModel => ({
+export const serializeEvent = (event: IEvent): IEventApi => ({
   title: event.title,
   description: event.description,
   location: event.location,
-  fromDate: stringifyDateTime(event.start),
-  toDate: stringifyDateTime(event.end),
-  responsibleEmployee: 1296,
+  startDate: event.start,
+  endDate: event.end,
+  openForRegistrationDate: event.openForRegistration,
+  organizerEmail: '1296',
+});
+
+export const toEvent = (event: WithId<IEventApi>): WithId<IEvent> => {
+  return {
+    ...event,
+    start: event.startDate,
+    end: event.endDate,
+    openForRegistration: event.openForRegistrationDate,
+  };
+};
+
+export const toEditEvent = (event: IEvent): IEditEvent => ({
```

Comment:
Kan ikkje vi automatisk generere ein EditEvent basert på ein Event ved å stringifye den også parse den? Høres kanskje stupid ut, men da har vi allereie denne funksjonaliteten "gratis", og vi slepper å passe på at vi er self-consistent. No er det to forskjellige måter å gjere ting på, som kan resultere i to forskjellige resultater.